client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/style.scss

```diff
@@ -1,3 +1,46 @@
 .site-content__mobile-view {
 	margin-bottom: 1.5rem;
 }
+
+.edit-all__bulk-actions {
+	margin: 10px;
+	padding-right: 5px;
+}
+.edit-all__bulk-actions button {
+	border: 1px solid var(--color-neutral-20);
+}
+button.edit-all__bulk-actions-close {
+	border: none;
+	padding: 4px 8px 4px 10px;
+}
+
+th.edit-all-buttons {
+	text-align: right;
```

Comment:
Could we use RTL-aware style attributes everywhere on this file? 

text-align: start/end